### PR TITLE
NW-6 | AREEB-SATTAR | REACT| React Module Project - CustomerProfile | WEEK-3

### PR DESCRIPTION
**Pull request by areebsattar**
## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/CustomerProfile/CustomerProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    );
+  }
+  return (
+    <tr className="customer-row">
+          <td colSpan={2}>{customerProfileData.phoneNumber}</td>
+          <td colSpan={2} className={customerProfileData.vip ? "vip-customer": ""}>
+          {customerProfileData.vip ? `VIP Customer`: `Regular Customer`}
+          </td>
+          <td>ID: {customerProfileData.id}</td>
+    </tr>
```
Comment (Ara225) [inline src/components/CustomerProfile/CustomerProfile.jsx:29]: Looks great! One thought, long names can get really annoying over time - naming is hard!

**Review by Ara225 — COMMENTED** [review]
Looking good!

File: src/components/CustomerProfile/CustomerProfile.test.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    render(<CustomerProfile id={1} />);
+
+    await screen.findByText("+44 1632 960185");
+
+    const phoneNumber = screen.getByText("+44 1632 960185");
+    expect(phoneNumber).toBeInTheDocument();
+    expect(screen.getByText("VIP Customer")).toBeInTheDocument();
+    expect(screen.getByText("ID: 1")).toBeInTheDocument();
+  });
```
[the remark this answers is not in the document]
Comment (areebsattar) [reply src/components/CustomerProfile/CustomerProfile.test.jsx:30]: Thank you, I tried looking up a few examples and implementing them. I still believe as a team we need more learning as it looks like I am the only one writing these

File: src/components/CustomerProfile/CustomerProfile.jsx (hunk 1 above)
Comment (areebsattar) [reply src/components/CustomerProfile/CustomerProfile.jsx:29]: I will try to come up with something better once I the project is complete

**Review by Sabella-8 — APPROVED** [review]

**Review by Sabella-8 — APPROVED** [review]

**Review by Sabella-8 — APPROVED** [review]
Looks good

**Pull request merged**